server/src/main/java/io/druid/segment/indexing/granularity/ArbitraryGranularitySpec.java

```diff
@@ -46,34 +45,25 @@
   private final TreeSet<Interval> intervals;
   private final QueryGranularity queryGranularity;
   private final Boolean rollup;
-  private final String timezone;
 
   @JsonCreator
   public ArbitraryGranularitySpec(
       @JsonProperty("queryGranularity") QueryGranularity queryGranularity,
       @JsonProperty("rollup") Boolean rollup,
-      @JsonProperty("intervals") List<Interval> inputIntervals,
-      @JsonProperty("timezone") String timezone
```

Comment:
Do you mean that another PR should be done before 0.10.0 to restore backwards compatibility?